packages/node/src/integrations/tracing/vercelai/instrumentation.ts

```diff
@@ -132,8 +138,21 @@ export class SentryVercelAiInstrumentation extends InstrumentationBase {
           recordOutputs,
         };
 
-        // @ts-expect-error we know that the method exists
-        return originalMethod.apply(this, args);
+        return handleCallbackErrors(
+          () => {
+            // @ts-expect-error we know that the method exists
+            return originalMethod.apply(this, args);
+          },
+          error => {
+            // This error bubbles up to unhandledrejection handler (if not handled before),
+            // where we do not know the active span anymore
+            // So to circumvent this, we set the active span on the error object
+            // which is picked up by the unhandledrejection handler
+            if (error && typeof error === 'object') {
+              addNonEnumerableProperty(error, '_sentry_active_span', getActiveSpan());
```

Comment:
fwiw `addNonEnumerableProperty` is really slow, but I guess in error codepaths this is fine.